save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/Config.kt

```diff
@@ -57,7 +58,7 @@ fun createConfigFromArgs(args: Array<String>): SaveConfig {
             }
     } catch (e: IOException) {
         logError("Unable to read properties file $propertiesFileName: ${e.message}")
-        emptyMap()  // todo exit with exit code?
+        exitProcess(ExitCodes.GENERAL_ERROR.code)
```

Comment:
I think this will not be the only place, where we should have this.
I guess that we need to unify it with logError into one function

Reply:
Added